test/withFx.test.js

```diff
@@ -33,7 +33,8 @@ describe("withFx", () => {
         exit: () => {
           done()
         }
-      }
+      },
+      Function.prototype
```

Comment:
What about writing, at the top of the file / module scope, `var functionPrototype = Function.prototype`?

...`fnProto`, `funcProto` sound good to me as well.

Reply:
@okwolf

Reply:
I've also seen folks define an alias of this as noop, since you're using it as a noop function.

Or we could make the view function optional again? 😉

Reply:
Certainly. In that case:

```js
var noop = function() {}
```

...might even be fewer bytes.

Reply:
What would be the point of making the view optional?

Reply:
? @jorgebucaran What would be the point of making the view optional?

Using Hyperapp for headless state management without having to create a fork.

Reply:
The container is already optional.

Reply:
> @jorgebucaran The container is optional, allowing you to use or test Hyperapp "headless", but what would be the point use case of making the view optional?

I guess that would depend on the use case but since we're not trying to advertise this project for standalone state management, it may only be an issue for the tests.

Reply:
@okwolf I reecently changed the docs to make known you can use HA "headless" by not using a container. But I still don't understand why you wouldn't want a view function. How are you going to get notified of state updates then?

Reply:
> @jorgebucaran How are you going to get notified of state updates then?

If you're writing an API, you may not need to be. You just need to get the current state (with actions) anytime a GET type request comes in, and fire update actions with POST/PUT/PATCH type requests.

Reply:
Using HA on the server as a state manager. That's another interesting use case I didn't think of. 

Thanks.

Reply:
I mean, I'm not going to open HA to that, yet, but it sounds cool.